core/src/com/etheller/warsmash/parsers/jass/Jass2.java

```diff
@@ -989,9 +989,7 @@ public boolean call(final CUnit unit) {
 										// TODO the trigger scope for evaluation here might need to be a clean one?
 										group.add(unit);
 										this.count++;
-										if (this.count >= countLimit) {
-											return true;
-										}
+                                        return this.count >= countLimit;
```

Comment:
![image](https://user-images.githubusercontent.com/5471783/165432080-23534eeb-b351-4d40-9ed2-aa78f9cb702f.png)
My code preview is displaying this return statement as being indented incorrectly. Are you using a different indentation paradigm than the rest of the Warsmash codebase? If so, shouldn't we either change the entire codebase, or change none of it?

Reply:
Hmm, strange, I see the same on github code preview, but in IDE I see this:
![image](https://user-images.githubusercontent.com/30243408/165637742-643b6751-7d12-4369-b710-6b0047d71d56.png)
Can you please try also in your IDE?